### ISSUE TITLE
"Unable to add. The user already exists". BUT It's not exists. What am I doing wrong? 

### ISSUE THREAD

**Issue by CaptainLoop** [pluginsGLPI/datainjection#242]
Hello!

Can you help me please with this? What I am doing wrong?
I try to import users from CSV, but only gives me errors. **Unable to add. The user already exists.**

![ErrorUsers](https://user-images.githubusercontent.com/13140552/129248756-3679e5a5-a478-44eb-bc47-8e00b91a1502.PNG)

If I go to Users, there is only the defaults. 
![onlyfew](https://user-images.githubusercontent.com/13140552/129248763-6cb4b2b4-7cb0-4f68-8a27-7cbda0a08061.PNG)

If I go to the Trashbin there is nothing.
![nothing](https://user-images.githubusercontent.com/13140552/129248759-d0446a11-553a-4bd6-b312-595f13cd84ff.PNG)

And I did not import them accidently to other "Entities".
![nothingagien](https://user-images.githubusercontent.com/13140552/129248762-0bdf89ee-8b8d-402b-8bce-c881a1204e4e.PNG)

**Comment by muzzol** [comment]
I'm facing the same problem and as I see the issue marked as completed I guess it was a bug but I don't know what to do.

My mapping is very simple: Name,Surname,email,comments

how can I debug the error?

**Comment by muzzol** [comment]
Ok, I've found the problem and I leave it here just in case someone gets stuck in the same situation.

Username mapping is mandatory. I was using email as username but I wasn't assigning it to username and the plugin doesn't allow to use one field for more than one mapping.
I solved recreating my fields from this: Name,Surname,email,comments
to this: email,name,surname,email,comments

and first email is mapped to username.

I hope this helps someone.
